shared/js/smart-screen/card_manager.js

```diff
@@ -671,6 +671,13 @@
     findCardFromCardList: function cm_findCardFromCardList(query) {
       var found;
       this._cardList.some(function(card) {
+        if (card instanceof Folder) {
+          found = card.findCard(query);
```

Comment:
I had checked that there are two places has `findCard`. If we only patch this place, it may have similar issue at folder.js. I know we don't have the use case that folder contains folder. But we may have it in the future. So, I would suggest:

1. we file another bug to move `findCard` as a shared library and use mixed in or shared library to call them.
2. we patch folder.js at the same time, although we don't need it right now.